src/main/resources/schemas/master_schema.json

```diff
@@ -881,7 +881,7 @@
                                               "title": "Information about linked cases",
                                               "type": "array",
                                               "items": {
-                                                "id": {
+                                                "caseId": {
                                                 "$id": "#root/courtLists/courtList/courtRoom/session/sittings/items/hearing/case/caseLinked/id",
```

Comment:
Think the indentation from line $id onwards is not right. A bit hard to read. I can see it has always been like that. Should be indented a bit more. Since you are changing that bit, can you update that please?

Reply:
fixed.